examples/rpc/README.md

```diff
@@ -0,0 +1,33 @@
+# RPC Demo
+
+A very simple example to show peer to peer communication with Godot's high level 
+
+multiplayer API calls.
+
+
+## Setup
+
+1. Build the library:
+
+    `cargo build`
+
+2. Import the Godot project in the Godot editor.
+
+3. Open the `Server.tscn` scene and start the current scene with `F6`.
+
+4. Start a second instance of the Godot editor.
+
+5. In the `Remote Port` in the Debug settings of the editor (e.g. 6012).
+
+    The two editor instances need to have different debug ports so that they can
+    
+    connect to their own running scenes. Needed to show the godot print messages.
+
+    `Editor > Editor Settings > Network > Debug`

```

Comment:
Extra newlines should be removed so the lines aren't interpreted as separate paragraphs.